src/components/organisms/sidenav/index.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+
+import menuIcon from '_images/menu.png'
+import closeIcon from '_images/close.png'
+import cklLogo from '_images/ckl-logo.png'
+import settingsIcon from '_images/settings.png'
+import NavigationItems from '_molecules/navigation-items'
+import NavigationItem from '_atoms/navigation-item'
+import NavigationFooter from '_atoms/navigation-footer'
+import IconButton from '_atoms/icon-button'
+import Icon from '_atoms/icon'
+
+import styles from './styles.css'
+
+const Sidenav = ({ isOpen, onCloseClick }) => {
+  const headerIcon = iconURL => (
+    <IconButton iconURL={iconURL} className={styles.iconButton} onClick={onCloseClick} />
+  )
+
+  return (
+    <div
+      className={classNames(
+        styles.sidenav,
+        { [styles.opened]: isOpen },
+        { [styles.closed]: !isOpen }
+      )}
+    >
+      <nav>
+        <div className={styles.navHeaderMobileOnly}>{headerIcon(closeIcon)}</div>
+        <div className={styles.navHeaderDesktopOnly}>{headerIcon(menuIcon)}</div>
+        <NavigationItems>
```

Comment:
use `children` instead.

Reply:
It's a good practice but this component controls the style of the navigation depending on if it's open or not or if it's a mobile or desktop.